### PR TITLE
grammar fixes

### PR DESCRIPTION
**Pull request by neheb**
Signed-off-by: Rosen Penev <rosenp@gmail.com>

### CODE REVIEW

File: src/content/onlineservice/task_processor.cc [hunk 1]
```diff
@@ -144,7 +144,7 @@ std::deque<std::shared_ptr<GenericTask>> TaskProcessor::getTasklist()
     auto lock = threadRunner->lockGuard();
     auto tc = getCurrentTask();
 
-    // if there is no current task, then the queues are empty
+    // if there are no current task, then the queues are empty
```
Comment (KarlStraussberger) [inline src/content/onlineservice/task_processor.cc:147]: IMHO: Either _is no current task_ or _are no current tasks_

File: src/main.cc [hunk 2]
```diff
@@ -263,9 +263,9 @@ int main(int argc, char** argv, char** envp)
             }
 
             // set all uids, gids and add. groups
-// mac os x does this differently, setgid and setuid are basically doing the same
+// macOS does this differently, setgid and setuid are basically doing the same
 // as setresuid and setresgid on linux: setting all of real{u,g}id, effective{u,g}id and saved-set{u,g}id
-// Solaroid systems are likewise missing setresgid and setresuid
+// Solaris systems are likewise missing setresgid and setresuid
```
Comment (KarlStraussberger) [inline src/main.cc:268]: That's fine it means system that behave like Solaris.

File: src/util/thread_runner.h [hunk 3]
```diff
@@ -178,9 +178,9 @@ class ThreadRunner : public ThreadExecutor {
     void startThread() override
     {
 #if !defined(SOLARIS) && !defined(__HAIKU__)
-        // default scoping on solaroid systems is in fact PTHREAD_SCOPE_SYSTEM
-        // plus, setting PTHREAD_EXPLICIT_SCHED requires elevated priveleges
-        // while Haiku doesn't implements pthread_attr_setinheritsched yet
+        // default scoping on Solaris systems is in fact PTHREAD_SCOPE_SYSTEM
```
Comment (KarlStraussberger) [inline src/util/thread_runner.h:181]: See above

**Comment by neheb** [comment]
Fixed.

**Pull request merged**